content/License.txt

```diff
@@ -1,58 +0,0 @@
-
```

Comment:
Would you be okay if we update its contents to "This file has moved, please see [link to new place]" ?